src/test/java/com/fewlaps/quitnowcache/SetAndFindValuesTest.java

```diff
@@ -0,0 +1,63 @@
+package com.fewlaps.quitnowcache;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class SetAndFindValuesTest extends BaseTest {
+
+    QNCache<String> cache;
+    MockDateProvider dateProvider;
+
+    @Before
+    public void init() {
+        cache = new QNCacheBuilder().createQNCache();
+        dateProvider = new MockDateProvider();
+        cache.setDateProvider(dateProvider);
+    }
+
+    @Test
+    public void setAndFindisSave() {
+        cache.set(A_KEY, A_VALUE);
+        cache.set(ANOTHER_KEY, ANOTHER_VALUE, THREE_DAYS);
+        cache.clear();
+
+        assertTrue(0 == cache.findStartsWith(A_KEY).size());
```

Comment:
If we use `assertEquals` instead of `assertTrue`, when the test fails, JUnit will give a better message. For example: `expected 0, actual 3`.
With the proposed solution it will say: `expected true, actual false`.
Please, use assertEquals over assertTrue in these cases